src/business-logic/services/got-http-service.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpService } from './interfaces/http-service'
+import got from 'got'
+
+export class GotHttpService implements HttpService {
+
+  public post(url: string, body: unknown): unknown {
+    return got.post(url, body ?? undefined)
```

Comment:
Is the nullish coalescing necessary? And in this implementation I can't send `null` as the body.
Instead the body parameter could default to undefined. That way you don't have to specify it when calling it.

Reply:
I've made the change you suggested :-)